satellite-imagery.md

```diff
@@ -0,0 +1,136 @@
+Satellite Imagery Processing
+============================
+
+Who am I?
+---------
+
+I am [David Hoese](http://github.com/djhoese) and I work as a software
+developer at the Space Science and Engineering Center (SSEC) at the
+University of Wisconsin - Madison. In a nut shell my job is to create
+software that makes meteorological data more accessible to scientists and
+researchers. I am also a member of the open source
+[PyTroll](http://pytroll.github.io/) community where I act as a core
+developer on the SatPy library. I use SatPy in my SSEC projects called
+Polar2Grid and Geo2Grid which provide a simple command line interface on top
+of the features provided by SatPy.
+
+
+The Problem I'm trying to solve
+-------------------------------
+
+Satellite imagery data comes in many shapes, sizes, and formats. Atmospheric
```

Comment:
I find that online readers have short attention spans.

If you're willing, I recommend starting a paragraph like this with a single line summary, perhaps something like the following:

> I want to make it easy to analyze a variety of kinds of satellite imagery.  This is hard because ...

> After loading data we want to do a variety of complex analyses.  

You might also consider switching to bullet points when appropriate in order to break up the wall of text.